scripts/dev_metrics.py

```
@@ -231,10 +235,58 @@ def tier2(commits: list[dict], now: datetime) -> dict:
+# (`crates/<family>/ironclaw_*`, PROPOSAL §5): `find` finds no files, every
+# probe returns 0, and the report renders composition at 0% of a 0-line
+# codebase — a perfect score for a tool that measured nothing.
+# See docs/reborn/target-architecture/CHECKLIST.md WS10.
+sys.path.insert(
+    0,
+    str(pathlib.Path(__file__).resolve().parent / "ci" / "lib"),
+)
+from crate_tree import crate_directories, crate_directory  # noqa: E402
+
+
+@functools.lru_cache(maxsize=None)
```

Ruff (0.16.0)

[warning] Use `@functools.cache` instead of `@functools.lru_cache(maxsize=None)`

Rewrite with `@functools.cache

(UP033)